tests/generic_tests/04_noauth.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+# Assuming you are running this from a prepared virtual environment
+PROJECT_ROOT=$(cd "$(dirname $0)/../.." && pwd)
+cd $PROJECT_ROOT
+c="python osc_sdk/sdk.py"
+
+echo -n "$(basename $0): "
+
+# Non-auth calls should always work
+$c api ReadRegions --authentication-method=accesskey &> /dev/null || { echo "api:ReadRegion error 1"; exit 1; }
```

Comment:
I'm not sure forcing authentication-method to none when `--authentication-method=accesskey` is specified in parameter is a good, idea, as this could be use to test ak/sk.

so maybe we should do  `self.authentication_method = "none"` only when no `--authentication-method` are pass in parameter

Reply:
je valide

Reply:
lets fix that

Reply:
This is now fixed but outscale API and ICU still accept to respond to calls which do not requiere authentication even if  user provides `accesskey` or `login` authentication.